README.md

```diff
@@ -14,7 +14,12 @@ your machine.
 
 ## Installation
 
-Install this package using [Composer](https://getcomposer.org/):
+
+### PHAR
+The recommended method of installing this package is using a PHAR file. This is because installing using Composer can possibly cause dependency conflicts. You can download the most recent PHAR from the [Github Releases](https://github.com/leviy/release-tool/releases) page.
```

Comment:
There is actually a https://github.com/leviy/release-tool/releases/latest URL that redirects to the latest release.